### ISSUE TITLE
verified_buffer has no method 'equals' error

### ISSUE THREAD

**Issue by kencochrane** [keybase/libkeybase-js#3]
Installing the latest version of keybase gives me the following error (below) when trying to run some commands. Tracking down the issue, it looks like it is coming from this lib, and starting with 1.2.1, I tried 1.2.2 and 1.2.3 those also fail. If I revert to 1.2.0, it works fine. so it looks like something that was changed in between 1.2.0 and 1.2.1 broke this.

Not sure if this is just me, or a wider issue. I uninstalled keybase and keybase-installer and started from scratch and I still had the issue.

## Running on

```
keybase (keybase.io CLI) v0.8.2
- node.js v0.10.35
- gpg (GnuPG/MacGPG2) 2.0.27
- libgcrypt 1.6.3
Identifies as: 'keybase.io node.js client v0.8.2 darwin'
```

## ERROR

```
$ keybase status
/usr/local/lib/node_modules/keybase/node_modules/libkeybase/lib/sigchain/sigchain.js:577
    if (!verified_buffer.equals(unverified_buffer)) {
                         ^
TypeError: Object {"body":{"cryptocurrency":{"address":"xxx","type":"xxx"},"key":{"fingerprint":"xxx","host":"keybase.io","key_id":"xxx","uid":"xxx","username":"xxx"},"type":"cryptocurrency","version":1},"ctime":xxx,"expire_in":157680000,"prev":null,"seqno":1,"tag":"signature"} has no method 'equals'
    at exports.check_buffers_equal.check_buffers_equal (/usr/local/lib/node_modules/keybase/node_modules/libkeybase/lib/sigchain/sigchain.js:577:26)
    at /usr/local/lib/node_modules/keybase/node_modules/libkeybase/lib/sigchain/sigchain.js:374:23
    at /usr/local/lib/node_modules/keybase/node_modules/libkeybase/lib/sigchain/sigchain.js:378:23
    at iced.Deferrals.parent (/usr/local/lib/node_modules/keybase/node_modules/libkeybase/lib/sigchain/sigchain.js:365:30)
    at /usr/local/lib/node_modules/keybase/node_modules/libkeybase/lib/sigchain/sigchain.js:367:21
    at Deferrals.exports.Deferrals.Deferrals._call (/usr/local/lib/node_modules/keybase/node_modules/iced-runtime/lib/runtime.js:86:16)
    at /usr/local/lib/node_modules/keybase/node_modules/iced-runtime/lib/runtime.js:98:26
    at exports.trampoline.trampoline (/usr/local/lib/node_modules/keybase/node_modules/iced-runtime/lib/runtime.js:64:14)
    at Deferrals.exports.Deferrals.Deferrals._fulfill (/usr/local/lib/node_modules/keybase/node_modules/iced-runtime/lib/runtime.js:96:16)
    at ret (/usr/local/lib/node_modules/keybase/node_modules/iced-runtime/lib/runtime.js:29:18)
```

**Comment by maxtaco** [comment]
@oconnor663 maybe the earlier version of node lacks this method.

**Comment by maxtaco** [comment]
Indeed, [v0.10.35](https://nodejs.org/docs/v0.10.35/api/buffer.html) lacks this method; it was introduced in [v0.12](https://nodejs.org/api/buffer.html#buffer_buf_equals_otherbuffer).

**Comment by maxtaco** [comment]
Closed, I hope, in v0.8.3.  LMK if not...
